packages/orbit-components/src/common/tailwind/color.ts

```diff
@@ -0,0 +1,470 @@
+    blueDarker: "mm:text-blue-darker",
+    cloudLight: "mm:text-cloud-light",
+    cloudLightHover: "mm:text-cloud-light-hover",
+    cloudLightActive: "mm:text-cloud-light-active",
+    cloudNormal: "mm:text-cloud-normal",
+    cloudNormalHover: "mm:text-cloud-normal-hover",
+    cloudNormalActive: "mm:text-cloud-normal-active",
+    cloudDark: "mm:text-cloud-dark",
+    cloudDarkHover: "mm:text-cloud-dark-hover",
+    cloudDarkActive: "mm:text-cloud-dark-active",
+    greenLight: "mm:text-green-light",
+    greenLightHover: "mm:text-green-light-hover",
+    greenLightActive: "mm:text-green-light-active",
+    greenNormal: "mm:text-green-normal",
+    greenNormalHover: "mm:text-green-normal-hover",
+    greenNormalActive: "mm:text-green-normal-active",
+    greenDark: "mm:text-green-dark",
+    greenDarkHover: "mm:text-green-dark-hover",
+    greenDarkActive: "mm:text-green-dark-active",
+    greenDarker: "mm:text-green-darker",
+    inkDark: "mm:text-ink-dark",
+    inkDarkHover: "mm:text-ink-dark-hover",
+    inkDarkActive: "mm:text-ink-dark-active",
+    inkLight: "mm:text-ink-light",
+    inkLightHover: "mm:text-ink-light-hover",
+    inkLightActive: "mm:text-ink-light-active",
+    inkNormal: "mm:text-ink-normal",
+    inkNormalHover: "mm:text-ink-normal-hover",
+    inkNormalActive: "mm:text-ink-normal-active",
+    orangeLight: "mm:text-orange-light",
+    orangeLightHover: "mm:text-orange-light-hover",
+    orangeLightActive: "mm:text-orange-light-active",
+    orangeNormal: "mm:text-orange-normal",
+    orangeNormalHover: "mm:text-orange-normal-hover",
+    orangeNormalActive: "mm:text-orange-normal-active",
+    orangeDark: "mm:text-orange-dark",
+    orangeDarkHover: "mm:text-orange-dark-hover",
+    orangeDarkActive: "mm:text-orange-dark-active",
+    orangeDarker: "mm:text-orange-darker",
+    productLight: "mm:text-product-light",
+    productLightHover: "mm:text-product-light-hover",
+    productLightActive: "mm:text-product-light-active",
+    productNormal: "mm:text-product-normal",
+    productNormalHover: "mm:text-product-normal-hover",
+    productNormalActive: "mm:text-product-normal-active",
+    productDark: "mm:text-product-dark",
+    productDarkHover: "mm:text-product-dark-hover",
+    productDarkActive: "mm:text-product-dark-active",
+    productDarker: "mm:text-product-darker",
+    redLight: "mm:text-red-light",
+    redLightHover: "mm:text-red-light-hover",
+    redLightActive: "mm:text-red-light-active",
+    redNormal: "mm:text-red-normal",
+    redNormalHover: "mm:text-red-normal-hover",
+    redNormalActive: "mm:text-red-normal-active",
+    redDark: "mm:text-red-dark",
+    redDarkHover: "mm:text-red-dark-hover",
+    redDarkActive: "mm:text-red-dark-active",
+    redDarker: "mm:text-red-darker",
+    socialFacebook: "mm:text-social-facebook",
+    socialFacebookHover: "mm:text-social-facebook-hover",
+    socialFacebookActive: "mm:text-social-facebook-active",
+    white: "mm:text-white-normal",
```

Comment:
this should be removed, I guess. we have only white-normal tokens, it should be as the same tokens, we do not have red, only redNormal and etc.

Reply:
We do have `white`. Maybe we should remove it, but it is being generated on the types file, so we have it.
It is defined [here](https://github.com/kiwicom/orbit/blob/setup-orbit-components-tailwind/packages/orbit-design-tokens/src/dictionary/definitions/global/palette/white.json#L4). Should it be removed? If so, on a different PR, no?

Reply:
Yes, you can remove it another PR :)